backend/package.json

```diff
@@ -6,27 +6,26 @@
   "scripts": {
     "build": "babel src/ -d dist/ --copy-files",
     "start": "node dist/",
-    "dev": "nodemon --exec babel-node src/ -e js,graphql",
-    "dev:debug": "nodemon --exec babel-node --inspect=0.0.0.0:9229 src/index.js -e js,graphql",
```

Comment:
> [<img alt="appinteractive" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/appinteractive) **Authored by [appinteractive](https://github.com/appinteractive)**
_<time datetime="2019-05-07T08:38:26Z" title="Tuesday, May 7th 2019, 10:38:26 am +02:00">May 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "babel src/ -d dist/ --copy-files",
     "start": "node dist/",
-    "dev": "nodemon --exec babel-node src/ -e js,graphql",
+    "dev": "nodemon --exec cross-env DISABLED_MIDDLEWARES=permissions babel-node src/ -e js,graphql",
```


Please do not deactivate the permissions in dev!!!!!!!!

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-15T14:47:47Z" title="Wednesday, May 15th 2019, 4:47:47 pm +02:00">May 15, 2019</time>_

---


@ogerly yes, please revert that one